main.tf

```diff
@@ -1,3 +1,28 @@
-/********************************************************************
-This file is used to implement the ROOT module.
-*********************************************************************/
+##############################################################################
+# terraform-ibm-scc-workload-protection
+#
+# Creates Security and Compliance Center Workload Protection
+##############################################################################
+
+##############################################################################
+# SCC WP
+##############################################################################
+
+resource "ibm_resource_instance" "scc_wp" {
+  name              = "${var.unique_name}-scc-wp"
+  resource_group_id = var.resource_group_id
+  service           = "sysdig-secure"
+  plan              = var.scc_wp_service_plan
+  location          = var.region
+  tags              = var.scc_wp_tags
+}
+
+##############################################################################
+# SCC WP Instance Key
+##############################################################################
+
+resource "ibm_resource_key" "scc_wp_key" {
+  name                 = "${var.unique_name}-resource-key"
+  role                 = var.scc_wp_key_role
+  resource_instance_id = ibm_resource_instance.scc_wp.id
```

Comment:
We need to support being able to create multiple service keys. Please take same approach as all ICD modules where we take in a map. For example https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/blob/f6227ddaa978cd2b1f9588039315b3b523c016e0/variables.tf#L65